Alignment/OfflineValidation/macros/PlotAlignmentValidation.C

```diff
@@ -429,12 +479,13 @@ void PlotAlignmentValidation::plotSS( const std::string& options, const std::str
       continue;
 
     // Skips plotting too high layers
-    if (plotLayerN > numberOfLayers[iSubDet-1]) {
+    if (plotLayerN > numberOfLayers(1, iSubDet)) {
```

Comment:
@hroskes not sure I fully understand the code, but it looks like hard-coded phase-1